packages/virtual/mediacenter/package.mk

```diff
@@ -46,4 +46,11 @@ if [ "$MEDIACENTER" = "kodi" ]; then
     PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET peripheral.joystick"
   fi
 
+  get_graphicdrivers
+  for drv in $GRAPHIC_DRIVERS; do
+    if [ "$drv" = "i915" -o "$drv" = "i965" ]; then
+      PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET intel-vaapi-driver"
+    fi
+  done
```

Comment:
Won't this add `intel-vaapi-driver` potentially twice (once for `i915`, and again for `i965`)?

This might be better:
```
  get_graphicdrivers
  if listcontains "$GRAPHIC_DRIVERS" "i915" || \
     listcontains "$GRAPHIC_DRIVERS" "i965"; then
    PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET intel-vaapi-driver"
  fi
```

The following should also work:
```
  get_graphicdrivers
  if listcontains "$GRAPHIC_DRIVERS" "(i915|i965)"; then
    PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET intel-vaapi-driver"
  fi
```
as `listcontains` is evaluating regular expressions.